### PR TITLE
Update fp-003-uris.md

### PR DESCRIPTION
**Pull request by wdduncan**
Removed Note section.  
Removed broken NCI link. 
TODO:  
Insert Summary and Purpose sections at top to match other principles layout.

### CODE REVIEW

File: principles/fp-003-uris.md [hunk 1]
```diff
@@ -58,8 +53,5 @@ Counter-Examples
 
 -   There are systems that use alphanumeric id's. This should be
     discouraged, especially as these have semantic content.
--   An example is this NCI class: 'Chemotherapy Regimen or Agent
-    Combination':
-    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#Chemotherapy_Regimen>
```
Comment (matentzn) [inline principles/fp-003-uris.md:63]: Why would you remove this, this is not really a URL - its a IRI, so it does not have to resolve to anything, right?

Comment (wdduncan) [reply principles/fp-003-uris.md:63]: We thought it would be better if it referenced a resolvable resource.

Comment (matentzn) [reply principles/fp-003-uris.md:63]: Ok cool, if you discussed this with the rest, I am in. Thanks for clarification!

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
@wdduncan you indicated that this pull request is not done? Otherwise it can be merged :)

**Pull request merged**